packages/sit-onyx/src/components/TestInput/TestInput.spec.tsx

```diff
@@ -1,16 +1,22 @@
-import { expect, test } from "@playwright/experimental-ct-vue";
+import { expect, test } from "../../playwright-axe";
```

Comment:
Can we enforce that the import always comes from the fixture?

Reply:
Probably only with a custom eslint rule, let me check that

Reply:
Done :)